fakturogen-invoice/src/main/java/pl/fakturogen/invoice/web/dto/CustomerDTO.java

```diff
@@ -1,4 +1,35 @@
 package pl.fakturogen.invoice.web.dto;
 
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import pl.fakturogen.invoice.dao.entity.CustomerType;
+
+import javax.validation.constraints.NotBlank;
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
 public class CustomerDTO {
+
+    private Long id;
+
+    private Long idExternalApi;
+
+    @NotBlank
+    private String name;
+
+    private String customerTaxNumber;
+
+    private String customerCode;
+
+    private String mail;
+
+    private String phoneNumber;
+
+    private CustomerType customerType;
```

Comment:
I think that here you should probably have CustomerTypeDTO instead of CustomerType, as in DTO we should use other DTO's not entities.